src/Sylius/Bundle/ApiBundle/Helpers/PriceTransformer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Sylius\Bundle\ApiBundle\Helpers;
+
+
+class PriceTransformer
+{
+    protected const PRICE_DIVIDER = 100;
+    protected const PRICE_MULTIPLIER = 100;
+
+    public static function centimesToUnitsForRequiredFields(int $value): float
+    {
+        if ($value === 0) {
+            return 0;
+        }
+
+        return (float)bcdiv($value, self::PRICE_DIVIDER, 2);
+    }
+
+    public static function centimesToUnitsForNullableFields(?int $value): ?float
+    {
+        if ($value === null || $value === 0) {
+            return $value;
+        }
+
+        return (float)bcdiv($value, self::PRICE_DIVIDER, 2);
+    }
```

Comment:
An example that describe that will be in case when value is null, and you are type casting the returned value 
http://sandbox.onlinephpfunctions.com/code/addc8521b070be55d39ae828f138d8b5bb57be69 in this case, we have redundant method